src/LatteContext/Collector/NetteApplicationUIPresenterCollector.php

```diff
@@ -6,7 +6,7 @@
 
 use Efabrica\PHPStanLatte\LatteContext\CollectedData\CollectedMethodCall;
 use Efabrica\PHPStanLatte\LatteContext\CollectedData\CollectedTemplateRender;
-use Efabrica\PHPStanLatte\LatteTemplateResolver\NetteApplicationUIPresenter;
+use Efabrica\PHPStanLatte\LatteTemplateResolver\Nette\NetteApplicationUIPresenter;
```

Comment:
What else namespace it could be?

Reply:
Nothing right now. But it felt a bit cluttered to me so I separated it to sub namespace.